gui/elm/Window/Tray/Settings.elm

```diff
@@ -93,6 +94,9 @@ update msg model =
         CloseApp ->
             ( { model | busyQuitting = True }, Ports.closeApp () )
 
+        Sync ->
+            ( model, Ports.manualStartSync () )
```

Comment:
We don't have any feedback to the user? After he/she has clicked on the button, he/she won't see anything that can tell him/her that his/her action has been taken in account?

Reply:
Now we have 7410d91c3b3531c723c49ff5f8468ec84b3f23fc